app/views/content/steps-to-become-a-teacher/_step_4_find_the_right_training.html.erb

```diff
@@ -1,5 +1,5 @@
-<p>Explore your options, find out <a href="/ways-to-train">how to train to
-  teach primary or secondary</a> (for example <a href="/what-is-a-pgce">a PGCE</a> or directly in a school).</p>
+<p>Explore your options, find out how to get a teaching qualification and <a href="/ways-to-train">how to train to
```

Comment:
Don't feel like this sentence works? Bit complicated and unclear, as if getting a teaching qualification is separate from how you train to teach primary or secondary. How about:

'There are different ways that you can gain your teaching qualification (QTS). This could be through an undergraduate teaching degree, a [PGCE](/what-is-a-pgce) or directly in a school. Find out about the [teaching courses available to teach primary or secondary](/ways-to-train).'